src/um/dwmapi.rs

```diff
@@ -53,7 +53,12 @@ extern "system" {
     // pub fn DwmRegisterThumbnail();
     // pub fn DwmSetDxFrameDuration();
     // pub fn DwmSetPresentParameters();
-    // pub fn DwmSetWindowAttribute();
+    pub fn DwmSetWindowAttribute(
+        hWnd: HWND,
+        dwAttribute: DWORD,
+        pvAttribute: LPVOID,
```

Comment:
[MSDN says this is LPCVOID.](https://msdn.microsoft.com/en-us/library/windows/desktop/aa969524)

Reply:
My mistake. Fixed.